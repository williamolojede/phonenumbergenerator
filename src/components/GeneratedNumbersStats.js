import React from 'react'

export default function GeneratedNumberStats({ count, min, max }) {
  return (
    <ul className="list-reset">
      <li className="text-center">
        <p className="text-grey-dark">MIN</p>
        <p className="mt-3 text-xl font-bold uppercase text-grey-darker">0{min}</p>
      </li>
      <li className="text-center mt-3">
        <p className="text-grey-dark">MAX</p>
        <p className="mt-3 text-xl font-bold uppercase text-grey-darker">0{max}</p>
      </li>
      <li className="text-center mt-3">
        <p className="text-grey-dark">TOTAL NUMBERS GENERATED</p>
        <p className="mt-3 text-xl font-bold uppercase text-grey-darker">{count}</p>
      </li>
    </ul>
  )
}
