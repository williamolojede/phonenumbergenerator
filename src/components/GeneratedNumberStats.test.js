import React from 'react'
import { mount } from 'enzyme';

import GeneratedNumbersStats from './GeneratedNumbersStats'

it('renders without crashing', () => {
  const wrapper = mount(<GeneratedNumbersStats max={199046623} min={100494045} count={500}/>);

  expect(wrapper).toMatchSnapshot();
});