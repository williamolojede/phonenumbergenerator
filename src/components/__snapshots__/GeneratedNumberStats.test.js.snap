// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<GeneratedNumberStats
  count={500}
  max={199046623}
  min={100494045}
>
  <ul
    className="list-reset"
  >
    <li
      className="text-center"
    >
      <p
        className="text-grey-dark"
      >
        MIN
      </p>
      <p
        className="mt-3 text-xl font-bold uppercase text-grey-darker"
      >
        0
        100494045
      </p>
    </li>
    <li
      className="text-center mt-3"
    >
      <p
        className="text-grey-dark"
      >
        MAX
      </p>
      <p
        className="mt-3 text-xl font-bold uppercase text-grey-darker"
      >
        0
        199046623
      </p>
    </li>
    <li
      className="text-center mt-3"
    >
      <p
        className="text-grey-dark"
      >
        TOTAL NUMBERS GENERATED
      </p>
      <p
        className="mt-3 text-xl font-bold uppercase text-grey-darker"
      >
        500
      </p>
    </li>
  </ul>
</GeneratedNumberStats>
`;
